submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/security/web/SecurityMapper.kt

```diff
@@ -28,9 +26,8 @@ class SecurityMapper {
 
     private fun getAllow(user: SecurityUser): List<String> {
         val accessTags = ArrayList<String>()
-        accessTags.add("~" + user.email)
-        accessTags.add("#" + user.id)
```

Comment:
Are you sure we don't need this anymore?

Reply:
no we are not

Reply:
Feel free to delete the "~" and the user id (including #). The front end only needs to match the accessTags in the submission with the allow list and already ignores # and ~.